lib/recaptcha/helpers.rb

```diff
@@ -235,13 +237,24 @@ def self.to_message(_key, default)
       options[:inline_script] != false
     end
 
-    private_class_method def self.recaptcha_v3_define_default_callback(callback)
-      <<-HTML
-          var #{callback} = function(id, token) {
-            var element = document.getElementById(id);
-            element.value = token;
-          }
-      HTML
+    private_class_method def self.recaptcha_v3_define_default_callback(callback, options)
+      if options[:ignore_no_element]
+        <<-HTML
+            var #{callback} = function(id, token) {
+              var element = document.getElementById(id);
+              if (element !== null) {
```

Comment:
what are you thoughts on either:
- always checking and not having an option at all
- using a single code block that inlines the option
?

```suggestion
              if (element !== null || #{!options[:ignore_no_element]}) {
```

Reply:
I think having the option as false by default will help us catch other edge cases, but we could consider making it true by default in the future. Using a single code block would simplify the code, so I'll make the changes accordingly. Let me know if you have any further suggestions or if you'd like me to rework the changes into the PR.

Reply:
all good, I'll merge once it's a single code block 👍